src/login/logind.conf.in

```diff
@@ -29,6 +29,7 @@
 #HandleLidSwitchExternalPower=suspend
 #HandleLidSwitchDocked=ignore
 #HandleRebootKey=reboot
+#HandleRebootKeyLongPress=ignore
```

Comment:
i wonder if we shouldn't default this to "poweroff". i.e. if we have the concept we might as well make use of it by default, and it appears reasonably to me for devices that just have one key to make it available for both operations.

Reply:
Then there will be a slight change in the default behaviour, because now reset action is triggered on the key press, and with long press enabled it will be triggered on the key release.

Reply:
seems ok. maybe mention this in a new NEWS file entry, but it's so minor I think it's fine.

Reply:
OK